lua/nvim_lsp/pyls_ms.lua

```diff
@@ -0,0 +1,140 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+
+local name = "pyls_ms"
+
+local function get_active_interpreter()
+  local f = io.popen("which python 2>&1")
```

Comment:
instead of `popen` with `2>&1`, could use the vim function `exepath`:

    local f = vim.fn.exepath('python')

Reply:
Fixed, thanks for pointing me to the function